tests/check_odov3.sh

```diff
@@ -39,8 +39,7 @@ ginkgo run --procs 2 \
   --skip="stack: java-wildfly" \
   --skip="stack: java-openliberty" \
   --skip="stack: java-websphereliberty" \
-  --skip="stack: java-quarkus version: 1.3.0 starter: community" \
-  --skip="stack: java-quarkus version: 1.4.0 starter: community" \
+  --skip="stack: java-quarkus" \
```

Comment:
Should we also change `check_odov2.sh` to match this? https://github.com/devfile/registry/blob/38d76c04a9c8b22e74814fac34b71fc91ca14186/tests/check_odov2.sh#L209

Reply:
Actually, I believe `check_odov2.sh` is already skipping all `java-quarkus` stacks, whereas `check_odov3.sh` was skipping just the `community` starter until now.

Reply:
> Actually, I believe `check_odov2.sh` is already skipping all `java-quarkus` stacks, whereas `check_odov3.sh` was skipping just the `community` starter until now.

@Jdubrick Its just the versions that are separated in `check_odov2.sh`, it works for now but will need to update this statement every time we bump a new version of the stack until the issues that require skipping are resolved.